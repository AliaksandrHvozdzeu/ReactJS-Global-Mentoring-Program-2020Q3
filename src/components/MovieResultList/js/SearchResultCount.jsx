import React from 'react';
import PropTypes from 'prop-types';

const SearchResultCount = ({ count }) => (
  <div className="movie-result-title">
    <p className="movie-result-text">
      <span className="movie-result-count">{count}</span>
      movies found
    </p>
  </div>
);

SearchResultCount.propTypes = {
  count: PropTypes.number.isRequired,
};

export default SearchResultCount;
