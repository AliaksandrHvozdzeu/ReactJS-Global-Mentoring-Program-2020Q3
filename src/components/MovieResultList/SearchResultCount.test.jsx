import React from 'react';
import rerender from 'react-test-renderer';
import SearchResultCount from './js/SearchResultCount';

describe('when SearchResultCount', () => {

  test('then snapshot created', () => {
    const count = 200;
    const component = rerender.create(<SearchResultCount count={count}/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
