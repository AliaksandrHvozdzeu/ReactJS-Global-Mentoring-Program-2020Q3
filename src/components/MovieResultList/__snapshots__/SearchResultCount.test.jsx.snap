// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when SearchResultCount then snapshot created 1`] = `
<div
  className="movie-result-title"
>
  <p
    className="movie-result-text"
  >
    <span
      className="movie-result-count"
    >
      200
    </span>
    movies found
  </p>
</div>
`;
